backend/analytics_server/mhq/api/resources/code_resouces.py

```diff
@@ -172,3 +172,8 @@ def adapt_team_repos(team_repos: List[TeamRepos]) -> List[Dict[str, any]]:
         }
         for team_repo in team_repos
     ]
+
+def get_non_paginated_merged_without_review(prs: List[PullRequest]) -> List[Dict[str,any]]:
+    return {
+        "data": []
```

Comment:
I think this func has to be deleted ? since you are now using `get_non_paginated_pr_response`

Reply:
@samad-yar-khan  Was going to implement my own  before finding the` get_non_paginated_pr_response` , and if i use the `_filter_prs_merged_in_interval`, i would not need to query for merge state explicitly again as it will be handled by this method